release.sh

```diff
@@ -63,9 +59,16 @@ if [ "${VERSION_SEGMENT}" == "major" ]; then
 elif [ "${VERSION_SEGMENT}" == "minor" ]; then
     echo "Bumping the minor version..."
     bumpversion minor
-else
+elif [ "${VERSION_SEGMENT}" == "patch" ]; then
     echo "Bumping the patch version..."
     bumpversion patch
+else
+    echo "Using override version ${VERSION_SEGMENT}"
```

Comment:
Why change up the `else`, the idea is that if anything the user passes in is not major, minor, or patch default to patch.

The alternative would be to do an `if` check above with with an `usage; exit 1` if it isn't valid.

Either way, definitely want the choice limited to major, minor, patch